src/components/Hangar/HangarComponent.js

```diff
@@ -159,14 +190,18 @@ const Hangar = (props) => {
                       {accountTank.isPremium === true && <StarFill size={1} color="Orange" />}
                     </Stack>
                   </td>
-                  <td>{new Date(accountTank.lastBattleTime * 1000).toLocaleDateString()}</td>
-                  <td>{accountTank.battles}</td>
-                  <td>{`${accountTank.winrate}%`}</td>
-                  <td>{accountTank.avgDmg}</td>
-                  <td>{accountTank.coefFrag}</td>
-                  <td>{accountTank.percentRemainHP[0] === '-' ? '0.00%' : `${accountTank.percentRemainHP}%`}</td>
-                  <td>{accountTank.battlesForMaster === 0 ? '-' : accountTank.battlesForMaster}</td>
-                  <td>{accountTank.avgTimeInBattle}</td>
+                  <td>{new Date(lastSnapshot.lastBattleTime * 1000).toLocaleDateString()}</td>
+                  <td>{lastSnapshot.regular.battles}</td>
+                  <td>{`${((lastSnapshot.regular.wins / lastSnapshot.regular.battles) * 100).toFixed(2)}%`}</td>
+                  <td>{~~(lastSnapshot.regular.damageDealt / lastSnapshot.regular.battles)}</td>
+                  <td>{(lastSnapshot.regular.frags / lastSnapshot.regular.battles).toFixed(2)}</td>
+                  <td>{
+                    ((1 - (lastSnapshot.regular.damageReceived / lastSnapshot.regular.battles) / accountTank.hp) * 100).toFixed(2) === '-' ?
+                    '0.00%' :
+                    `${((1 - (lastSnapshot.regular.damageReceived / lastSnapshot.regular.battles) / accountTank.hp) * 100).toFixed(2)}%`
+                  }</td>
+                  <td>{~~(lastSnapshot.regular.battles / lastSnapshot.mastery.markOfMastery) === 0 ? '-' : ~~(lastSnapshot.regular.battles / lastSnapshot.mastery.markOfMastery) }</td>
+                  <td>{avgTimeInBattle}</td>
```

Comment:
Prev comment explain all
```suggestion
                  <td>{new Date(stats.lastBattleTime * 1000).toLocaleDateString()}</td>
                  <td>{stats.battles}</td>
                  <td>{`${stats.winrate}%`}</td>
                  <td>{stats.avgDmg}</td>
                  <td>{stats.coefFrag}</td>
                  <td>{
                     stats.percentRemainHP === '-' ?
                     '0.00%' :
                      `${stats.percentRemainHP}%`
                   }</td>
                  <td>{stats.battlesForMaster === 0 ? '-' : stats.battlesForMaster }</td>
```